utkdigitalinitiatives/TRACE
# address TRAC-1165: update Solr results' 'Collection' field mapping

** JIRA Ticket**: [TRAC-1165](https://jira.lib.utk.edu/browse/TRAC-1165)

# What does this Pull Request do?
This PR updates a field definition in one of the Solr metadata configuration pages.

# What's new?
Changed `RELS_EXT_hasModel_uri_ms` to `mods_relatedItem_series_titleInfo_title_s`.

# How should this be tested?
1. `vagrant destroy -f`
2. `git pull --all`
3. `vagrant up`
4. visit `http://localhost:8000`
5. perform either an empty search or 'Combined Loading' search.
6. a passing result is that results returned with a 'Collection' field should be displaying 'Graduate Thesis and Dissertations'.

# Interested parties
@utkdigitalinitiatives/digital-initiatives-developers
